src/components/Modal/components/CloseButton/CloseButton.tsx

```diff
@@ -11,15 +13,20 @@ export interface Props {
   onClick(): void;
 }
 
-export default function CloseButton({title = true, onClick}: Props) {
+type CombinedProps = Props & WithAppProviderProps;
+
+function CloseButton({title = true, onClick, polaris: {intl}}: CombinedProps) {
   const className = classNames(
     styles.CloseButton,
     !title && styles.withoutTitle,
   );
 
   return (
     <button onClick={onClick} className={className}>
+      <VisuallyHidden>{intl.translate('Polaris.Common.close')}</VisuallyHidden>
```

Comment:
Nothing wrong with `VisuallyHidden` here, but [our recommendation in the style guide](https://polaris.shopify.com/components/images-and-icons/icon) is to use the `Icon` component’s `accessibilityLabel` prop.

Reply:
I have fixed this up